### ISSUE TITLE
Why need sudo for libhidapi-libusb and not for libhidapi-hidraw?

### ISSUE THREAD

**Issue by FauthD** [libusb/hidapi#656]
Hi,
on Ubuntu 22.04 I need sudo for a test tool (Python bindings) when libhidapi-libusb is installed.
Enforcing  libhidapi-hidraw (by uninstalling libusb backend) allows my tool to run without sudo.
Note: I have access rights to the /dev/hidrawX without sudo.

Thanks!

**Comment by Youw** [comment]
From main [README](https://github.com/libusb/hidapi?tab=readme-ov-file#hidapi-has-four-back-ends):

> Note that you will need to install an udev rule file with your application for unprivileged users to be able to access HID devices with hidapi. Refer to the [69-hid.rules](https://github.com/libusb/hidapi/blob/master/udev/69-hid.rules) file in the udev directory for an example.

**Comment by FauthD** [comment]
That's what I did in first place, did develop USB stuff since 1996 (with Linux since 2004).
The devices use "root plugdev" (owner group).
`crw-rw---- 1 root plugdev 234, 0 Jan 15 22:13 /dev/hidraw0`

Tools using the HidRaw backend work fine without sudo.
Only the UsbLib backend needs sudo (unfortunatelly this is prioritized from Python bindings).
Assumption is some path in /sys/* needs root root. It is that many paths are used in enumeration, probably one of it is the problem.
Double checked the rights for the .so files, these are the same to each other.

And some lib (dfUsbLib) I wrote some years ago also works fine without sudo (always worked), but that only used libudev, not libhidraw.

Next days I will analyze further, just had the hope that someone already knows about this issue.
